c++/capnp.pc.in

```diff
@@ -1,11 +1,11 @@
 prefix=@prefix@
-exec_prefix=@exec_prefix@
```

Comment:
I missed this when reviewing this before, but removing these lines breaks these files. #289 fixes it.